app/views/spree/components/layout/_top_bar.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="top-bar">
   <%= render partial: 'spree/components/brand/logo' %>
   <div class="top-bar__search">
-    <%= render partial: 'spree/components/search/search_bar' %>
+    <%= render 'spree/components/search/search_bar' %>
```

Comment:
Why did you remove the `partial` keyword? 🤔

Reply:
I'm going to remove this change. I'll do this globally in another pr